Template/Variable/MatomoConfigurationVariable.php

```diff
@@ -42,9 +42,10 @@ public function hasAdvancedSettings()
     public function getParameters()
     {
         $idSite = Common::getRequestVar('idSite', 0, 'int');
+        $settings = new Settings();
+        $url = $settings->getAnalyticsUrl();
 
-        $url = SettingsPiwik::getPiwikUrl();
-        if (SettingsPiwik::isHttpsForced()) {
+        if ($settings->isHttpsForced()) {
```

Comment:
Not quite understanding this change @lockland 

`$url` seems unused, and `$settings->isHttpsForced` is just calling `SettingsPiwik::isHttpsForced()` so could as well keep the old code and keep it directly.

Reply:
I didn't understand why you thought $url is unused whereas this variable is processed on if/else statement below.
I've created a method isHttpsForced on SystemSettings because I want to remove SettingsPiwik dependency since, in my opinion, all settings should be handled by SystemSettings. As I have an object on the same context I just used it.